candle-transformers/Cargo.toml

```diff
@@ -24,6 +24,9 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 serde_plain = { workspace = true }
 tracing = { workspace = true }
+image = { workspace = true }
```

Comment:
It's on purpose that `candle-transformers` has a limited set of dependencies, in particular we don't want to have `transformers` etc so the relative bits of code should probably be moved to `candle-examples`.

Reply:
After refactor the candle-transformers/models/llava only has the most related thing about model, others are moved to candle-examples